crates/rust-analyzer/tests/heavy_tests/main.rs

```diff
@@ -707,3 +708,135 @@ pub fn foo(_input: TokenStream) -> TokenStream {
     let value = res.get("contents").unwrap().get("value").unwrap().to_string();
     assert_eq!(value, r#""```rust\nfoo::Bar\nfn bar()\n```""#)
 }
+
+#[test]
+fn build_rs_cfgs() {
```

Comment:
If that's not too onerous, I'd try to merge all `load_out_dirs_from_check` related tests into one. Heavy tests are slow, so it makes sense to optimize the number of times we interract with cargo.